datasets/guardian_authorship/guardian_authorship.py

```diff
@@ -0,0 +1,314 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace NLP Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This is an authorship attribution dataset based on the work of Stamatatos 2013. """
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import nlp
+
+_CITATION = """\
+@article{article,
```

Comment:
nit: could you make sure both citations have the same indentation ?

Reply:
Fixed.